### PR TITLE
Remove zoning-plan texts where not applicable

### PR DESCRIPTION
**Pull request by afjlambert**
Refactored the intropages a bit in the process

### CODE REVIEW

**Comment by afjlambert** [comment]
Don't merge please. We're awaiting editorial approval. Ready for review though.

**Comment by afjlambert** [comment]
I think the github action for file size doesn't work properly. On my machine it generates less code.
Schermafbeelding 2020-07-16 om 13 57 01
On the left is this branch. On the right is 'develop'

File: apps/client/src/intros/ZonnepanelenIntro.js [hunk 1]
```diff
@@ -1,41 +1,14 @@
...
-        <ListInsideOrderedList variant="bullet">
-          <ListItem>of het gebouw een monument is.</ListItem>
-          <ListItem>
-            of het gebouw in een beschermd stads- of dorpsgezicht ligt.
-          </ListItem>
-          <ListItem>welk bestemmingsplan er geldt.</ListItem>
-        </ListInsideOrderedList>
-      </ListItem>
-      <ListItem>
-        Wij stellen u een aantal vragen over het gebouw en de zonnepanelen of de
```
Comment (robinpiets) [inline apps/client/src/intros/ZonnepanelenIntro.js:28]: Now you're missing this text:
_Wij stellen u een aantal vragen over het gebouw **en de zonnepanelen of de zonneboiler.**_

File: apps/client/src/intros/DakraamIntro.js [hunk 2]
```diff
@@ -1,42 +1,17 @@
...
-        <ListInsideOrderedList variant="bullet">
-          <ListItem>of het gebouw een monument is.</ListItem>
-          <ListItem>
-            of het gebouw in een beschermd stads- of dorpsgezicht ligt.
-          </ListItem>
-          <ListItem>welk bestemmingsplan er geldt.</ListItem>
-        </ListInsideOrderedList>
-      </ListItem>
-      <ListItem>
-        Wij stellen u een aantal vragen over het gebouw en het dakraam.
```
Comment (robinpiets) [inline apps/client/src/intros/DakraamIntro.js:29]: Now you're missing this text:
_Wij stellen u een aantal vragen over het gebouw **en het dakraam**._

File: apps/client/src/intros/DakkapelIntro.js [hunk 3]
```diff
@@ -1,38 +1,12 @@
...
-        <ListInsideOrderedList variant="bullet">
-          <ListItem>of het gebouw een monument is.</ListItem>
-          <ListItem>
-            of het gebouw in een beschermd stads- of dorpsgezicht ligt.
-          </ListItem>
-          <ListItem>welk bestemmingsplan er geldt.</ListItem>
-        </ListInsideOrderedList>
-      </ListItem>
-      <ListItem>
-        Wij stellen u een aantal vragen over het gebouw en de dakkapel.
```
Comment (robinpiets) [inline apps/client/src/intros/DakkapelIntro.js:25]: Now you're missing this text:
_Wij stellen u een aantal vragen over het gebouw **en de dakkapel**._

**Review by robinpiets — CHANGES_REQUESTED** [review]
Nice refactor. Waiting on the team for final approval...

**Comment by robinpiets** [comment]
> I think the github action for file size doesn't work properly. On my machine it generates less code.
> Schermafbeelding 2020-07-16 om 13 57 01
> On the left is this branch. On the right is 'develop'

Weird that `apps/client/build/static/js/runtime-main.bceb6449.js` has `+2.98 kB` while on your terminal it says that they're equal. Did you build correctly?

**Comment by afjlambert** [comment]
There is actually a different runtime file because the `rename` script only fixes the chunks for some reason. It shows there is a new runtime file, what it doesn't show however is files that no longer exist. This is confusing.

File: apps/client/src/intros/ZonnepanelenIntro.js (hunk 1 above)
Comment (afjlambert) [reply apps/client/src/intros/ZonnepanelenIntro.js:28]: That's correct.

File: apps/client/src/intros/DakraamIntro.js (hunk 2 above)
Comment (afjlambert) [reply apps/client/src/intros/DakraamIntro.js:29]: That's correct.

File: apps/client/src/intros/DakkapelIntro.js (hunk 3 above)
Comment (afjlambert) [reply apps/client/src/intros/DakkapelIntro.js:25]: That's correct.

**Review by robinpiets — APPROVED** [review]
👍

**Pull request merged**